source/hardware/controllers/yaeltex_minimixxx.rst

```diff
@@ -0,0 +1,307 @@
+.. _yaeltex-minimixxx:
+
+Yaeltex MiniMixxx
+=================
+
+.. sectionauthor::
+  Owen Williams <owilliams at mixxx.org>
+
+The Yaeltex MiniMixxx is a compact, 2-deck DJ controller designed exclusively
+for Mixxx. Unlike most small form-factor DJ controllers which sacrifice
+important functions, the MiniMixxx provides access to all of the critical parts
+of DJing: full size EQ and quick FX knobs, pitch sliders, channel faders and
+track gain, playback controls including sync nudging, looping, an FX bank, 4
+hotcues per deck, 8 samplers, sync lock, keylock, library navigation, and master
+output settings. Button mode layers are intelligently laid out and road-tested
+for maximum flexibility during performance, with bold RGB colors indicating
+current modes. The visualization of channel volume and virtual record spinning
+indicators makes the MiniMixxx an eye-catching wonder in the DJ booth.
+
+The MiniMixxx can run on USB bus power. Using a separate power supply increases
+the brightness of the LEDs. The MiniMixxx does not have a built-in sound card.
+
+.. versionadded:: 2.4.0
+
+Compatibility
+-------------
+
+Controller
+~~~~~~~~~~
+
+The MiniMixxx is a USB class compliant MIDI device.  To work correctly with this
+mapping, you should load blah blah blah custom layout.
```

Comment:
?

Reply:
hahahahhaahhaa sorry.  The controller requires a specific config to be loaded on it, I have to attach that